sisyphus/istio_deployment/main.go

```diff
@@ -53,14 +53,12 @@ var (
 	catchFlakesByRun     = flag.Bool("catch_flakes_by_rerun", true, "whether to rerun failed jobs to detect flakyness")
 
 	protectedJobs = []string{
```

Comment:
you might able to query those from the github api directly instead of hard coding them

Reply:
Done